### DISCUSSION TITLE
panel login credentials from paymenter

### DISCUSSION THREAD

**Discussion by 1gzify** [Paymenter/Paymenter#236] [Ideas]
make it so people choose their username and password for ptero panel when they first sign up for paymenter site.

**Comment by Moreau741** [comment]
Customers, if ordered, will receive an email directly from Pterodactyl making it easy to choose your password. It is also not very convenient if Paymenter will provide passwords for Pterodactyl, the system currently used is convenient and efficient. 

With regard to username, you can also just login using your email at Pterodactyl.
